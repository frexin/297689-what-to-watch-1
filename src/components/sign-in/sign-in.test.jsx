import React from 'react';
import renderer from 'react-test-renderer';
import {BrowserRouter} from "react-router-dom";

import SignIn from "./sign-in";

it(`SignIn component loads correctly`, () => {
  const tree = renderer.create(<BrowserRouter><SignIn onFormSubmit={jest.fn()} /></BrowserRouter>).toJSON();

  expect(tree).toMatchSnapshot();
});
