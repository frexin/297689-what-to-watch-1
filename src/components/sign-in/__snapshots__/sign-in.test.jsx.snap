// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn component loads correctly 1`] = `null`;
